packages/fx-core/templates/plugins/resource/aad/auth/bot/js/sso/handleMessageExtensionQueryWithToken.js

```diff
@@ -0,0 +1,61 @@
+console.log("hello world js");
+import { TeamsActivityHandler, CardFactory, TurnContext } from "botbuilder";
+import {
+  handleMessageExtensionQueryWithToken,
+  TeamsFx,
+  createMicrosoftGraphClient,
+} from "@microsoft/teamsfx";
+import "isomorphic-fetch";
+
+export class TeamsBot extends TeamsActivityHandler {
```

Comment:
use module.exports instead of export in js?

Reply:
thanks, udpated